gui/src-tauri/src/main.rs

```diff
@@ -45,7 +51,8 @@ fn is_valid_path(path: &PathBuf) -> bool {
 
 fn get_launch_path(cli: Cli) -> Option<PathBuf> {
 	let mut path = cli.launch_from_path.unwrap_or_default();
-	if is_valid_path(&path) {
+	// The reason from all our problems was path being ""
```

Comment:
Could this comment explain what the problem was a little? It's a bit odd mentioning it caused problems but not *what* the problems were.

Reply:
i ended up changing this with an if let